adapters/adview/adviewtest/exemplary/native-app.json

```diff
@@ -0,0 +1,144 @@
+{
+    "mockBidRequest": {
+      "id": "some-request-id",
+      "device": {
+        "ua": "test-user-agent",
+        "ip": "123.123.123.123",
+	"ifa":"00000000-0000-0000-0000-000000000000",
```

Comment:
Please fix the inconsistent whitespace in this line.

Reply:
fixed.